src/azul/service/hca_response_v5.py

```diff
@@ -398,6 +398,17 @@ def make_protocols(self, entry):
             )
         ]
 
+    def make_dates(self, entry):
+        dates = []
+        for dates_ in entry['contents']['dates']:
+            dates.append({
+                **self._make_entity(dates_),
+                'aggregateLastModifiedDate': dates_['aggregate_last_modified_date'],
+                'aggregateSubmissionDate': dates_['aggregate_submission_date'],
+                'aggregateUpdateDate': dates_['aggregate_update_date'],
+            })
+        return dates
```

Comment:
Any reason not to use a list comprehension? Removes the need for a second variable name 
```suggestion
        return [
                {
                    **self._make_entity(dates_),
                    'aggregateLastModifiedDate': dates['aggregate_last_modified_date'],
                    'aggregateSubmissionDate': dates['aggregate_submission_date'],
                    'aggregateUpdateDate': dates['aggregate_update_date'],
                }
                for dates in entry['contents']['dates']
            )
```